modules/templates/article-static.xml

```diff
@@ -138,18 +137,8 @@ if (properties.copyright) {
 }
 
 // Render
-$(target).find('img').xiboImageRender(globalOptions);
-$(target).xiboLayoutScaler(globalOptions);
-
-var runOnVisible = function() {
-    $(target).xiboTextRender(properties, items);
-};
-
-if (xiboIC.checkVisible()) {
```

Comment:
Do we need an `onVisible` node to replace this?

Reply:
I forgot to update these files, I will update my branch ( and the PR ) with it